### ISSUE TITLE
[Feature Request] Accept comma separated list of flows or flow directories

### ISSUE THREAD

**Issue by tokou** [mobile-dev-inc/maestro#1990]
### Use case

Right now, the `flowFile` option of the CLI expects either a flow file path `flows/some_feature/some_flow.yml` or a flow directory path `flows/other_feature/`.

However I often run into cases where I want to run 2 sets of flow files (or more), but not the whole test suite.
Example: making sure a certain feature did not introduce regressions in another closely related feature.

Here's what happens when I do that:

```
> maestro test flows/some_feature/some_flow.yml,flows/other_feature/

Flow path does not exist: $SRC/e2e/flows/some_feature/some_flow.yml,flows/other_feature/
```

### Proposal

I'd like for `flowFile` to accept a comma-separated list of flow files or directories and to construct the flows to run as the union of all of those.

Basically: 

```
maestro test flows/some_feature/some_flow.yml,flows/other_feature/
```

Would be kinda like:

```
maestro test flows/some_feature/some_flow.yml

maestro test flows/other_feature/
```

### Anything else?

This is similar to the current behavior of `--device`

**Comment by bartekpacia** [comment]
Sounds good to me! PRs welcome!

**Comment by tokou** [comment]
Sure
